modules/generic-scala3/src/main/scala-3/pureconfig/generic/semiauto.scala

```diff
@@ -0,0 +1,6 @@
+package pureconfig.generic
+
+import pureconfig.generic.derivation.*
+
+object semiauto:
+  export HintsAwareConfigReaderDerivation.deriveReader
```

Comment:
I like that this module is kept as simple as possible, delegating all logic to an internal implementation 👍